lib/tasks/plantain.rake

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+require_relative "../../app/jobs/application_job"
+require_relative "../../app/jobs/index_job"
```

Comment:
I'm not certain why this was not available within the scope of the Rake Task - `Rails` was defined.